contracts/src/Dependencies/Constants.sol

```diff
@@ -28,6 +28,11 @@ uint128 constant MIN_INTEREST_RATE_CHANGE_PERIOD = 1 seconds; // prevents more t
 
 uint256 constant REDEMPTION_FEE_FLOOR = _1pct / 2; // 0.5%
 
+// For the debt / shares ratio to increase by a factor 1e9
```

Comment:
Yes this is the "natural" way. Though an inflation attack (as outlined by CS) would make it faster. But maybe we just document that in the docs rather than here.

Reply:
Good point. I added a small note there too, it doesn’t hurt.